### PR TITLE
JA: translation /ja/install

### PR DESCRIPTION
**Pull request by yukinagae**
I've translated the content under ja/install*.

Any feedback or comment would be appreciated :)

### CODE REVIEW

**Comment by lamberta** [comment]
Thanks you, Yuki, this is great.
Let me think a little on this because we need to figure out a good way to keep this section in sync with the en/ version. For tutorials and guides, these can be a bit out of date and still be useful. But for the /install section, it's very important we show the latest instructions, so we need to come up with a way t maintain this. Will see if we can come up with something.

**Comment by yukinagae** [comment]
@lamberta OK, that makes sense. Hope we can come up with some good ideas. At the moment, I can work on translating other sections I guess.

**Comment by lamberta** [comment]
Thanks, Yuki. I think this can be handled with a simple preprocessing step and a variable ot two. Did you come across anything besides `$TENSORFLOW_VERSION` that we would need to account for?

**Comment by yukinagae** [comment]
@lamberta Thanks for your reply, Billy. I think your idea is feasible and straightforward, and it must be the way we can go forward. One point I can add it is there may be the chances we have to change some of the content of installation, not only TensorFlow versions but also the issues and instructions regarding specific versions.

File: site/ja/install/docker.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+Dockerは、初めて実行する際に新しいTensorFlowのイメージをダウンロードします:
+
+<pre class="devsite-terminal devsite-click-to-copy prettyprint lang-bsh">
+docker run -it --rm tensorflow/tensorflow \
+   python -c "import tensorflow as tf; tf.enable_eager_execution(); print(tf.reduce_sum(tf.random_normal([1000, 1000])))"
+</pre>
+
+成功: TensorFlowがインストールできました。[チュートリアル](../tutorials)を読んで始めましょう。
+
+もう少しTensorFlowのDockerレシピをデモをしましょう。
```
[the remark this answers is not in the document]
Comment (yukinagae) [reply site/ja/install/docker.md]: Thanks for your comment, mattn. Which of the following sentences is better?

- 「もう少しTensorFlowのDockerのレシピを使ったデモをしましょう」
- 「もう少しTensorFlowのDockerの使い方を見てみましょう」

[the remark this answers is not in the document]
Comment (mattn) [reply site/ja/install/docker.md]: Original sentense use `demonstrate` so I prefer the second. (but this is just my optinion. Using "を" twice bothers me) :)

[the remark this answers is not in the document]
Comment (yukinagae) [reply site/ja/install/docker.md]: Got it!

**Comment by yukinagae** [comment]
@lamberta Any update on this?

**Comment by lamberta** [comment]
Looping in @sfujiwara, @ohtaman for ideas

File: site/ja/install/errors.md [hunk 2]
```diff
@@ -0,0 +1,311 @@
+# Build and install error messages
```
[the remark this answers is not in the document]
Comment (yukinagae) [reply site/ja/install/errors.md]: That's true, I'll do it.

**Comment by sfujiwara** [comment]
OK, I started to review them. But, I need some days because the diff is so large. Please wait for it.

**Comment by sfujiwara** [comment]
@yukinagae I did check `docker.md` and `gpu.md`. But, I'm sorry that it's a little bit tough to review this PR all at once. I think it's not good to take long time to merge this PR. Could you split this PR to each files?

@lamberta How do you think about the above idea?

**Comment by yukinagae** [comment]
@sfujiwara Sorry for the late reply. I've been away from this issue and just noticed your comment. 
I understand that this PR is too big for you to review. I'll try to split this PR to small PRs.

Thanks anyway!

**Comment by yukinagae** [comment]
@sfujiwara I made a small PR here, https://github.com/tensorflow/docs/pull/643.

If it is OK, I will make other small PRs as well such as gpu, c/go/java so on.

**Comment by yukinagae** [comment]
@sfujiwara I made small PRs below:

- gpu.md: https://github.com/tensorflow/docs/pull/679
- source.md: https://github.com/tensorflow/docs/pull/680
- source_rpi.md: https://github.com/tensorflow/docs/pull/681
- source_windows.md: https://github.com/tensorflow/docs/pull/682
- pip.html: https://github.com/tensorflow/docs/pull/683
- lang_c.md: https://github.com/tensorflow/docs/pull/684
- lang_go.md: https://github.com/tensorflow/docs/pull/685
- lang_java.md: https://github.com/tensorflow/docs/pull/686

**Comment by yukinagae** [comment]
@lamberta 

This PR is closed regarding your post below:

> TLDR: Please do not translate the install/ section or *.yaml files (_book.yaml/_toc.yaml/_index.yaml) for Japanese, Korean, or Chinese

https://groups.google.com/a/tensorflow.org/d/msg/docs-ja/f4Vh66EnPpo/2PgmziVrAgAJ
